### PR TITLE
Added support for batch trigger events

### PR DESCRIPTION
**Pull request by Tybot204**
Pusher supports sending batch events since version 1.1.0 of their Ruby gem. I added in an endpoint for the fake server so that Rails applications can make use of this in testing.

See the [API documentation](https://pusher.com/docs/rest_api#method-post-batch-events).

### CODE REVIEW

**Comment by tristandunn** [comment]
Awesome! Thanks for the work. Let me know when it's finished and I'll try to get it merged as soon as possible.

Also, I fixed the Rubocop issue in master if you want to rebase.

**Comment by Tybot204** [comment]
@tristandunn I think this is ready now. Couple other things to note:
1) The Pusher API docs say their endpoints for events and batch events allow "channel" (singular) to be used in place of "channels". I added a conditional assignment to both fake emitters to handle that case here.
2) My tests are all passing locally, yet 2 of them fail on Travis CI. Is this a configuration problem with Travis? It looks like capybara-webkit is having trouble connecting.

File: lib/pusher-fake/server/application.rb [hunk 1]
```diff
@@ -50,6 +52,29 @@ def self.events(request)
       end
       # rubocop:enable Style/RescueModifier
 
+      # Emit a batch event with data to the requested channel(s).
+      #
+      # @param [Rack::Request] request The HTTP request.
+      # @return [Hash] An empty hash.
+      #
+      # rubocop:disable Style/RescueModifier, Metrics/AbcSize
```
Comment (tristandunn) [inline lib/pusher-fake/server/application.rb]: I'm fine with ignoring the ABC size for now. I'll refactor the method out.

**Comment by tristandunn** [comment]
1. Nice, thanks.
2. Yeah, they fail on CI sometimes. I've just never got around to figuring out why. They usually pass if you re-run them once or twice.

Thanks again!

**Comment by tristandunn** [comment]
@Tybot204 Just released v1.7.0 with this in it, if you're looking to use it.

**Comment by Tybot204** [comment]
Awesome, thank you! Updating my gems right now.

**Pull request merged**